### PR TITLE
feature/v1-083/adaptive layout & menu + bugfix/v1-069

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
1. Adapted topbar for ipad mini.
2. Topbar for iphone with menu button.
3. Added mobile menu.
4. For mobile version added search and filter on courses pages.
5. Adaptive No Content component.
6. Fixed bug on v1-069, that appeared after adding menu squeeze.

### CODE REVIEW

File: frontend/src/components/Layout/MobileMenu/MobileMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                  />
+                </TabWrapper>
+              ),
+            )}
+          </MenuTabsWrapper>
+        </MenuTabs>
+        <SpaceHolder
+          onClick={(e) => {
+            e.stopPropagation();
+          }}
```
Comment (k-krasilnikova) [inline frontend/src/components/Layout/MobileMenu/MobileMenu.tsx:92]: Move it please to the separate function

File: frontend/src/components/Layout/MobileMenu/MobileMenuContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        pathname={pathname}
+        isTabHeader={TAB_HEADER}
+        isMobileMenuOpen={isMobileMenuOpen}
+        toggleMobileMenu={toggleMobileMenu}
+        firstName={firstName}
+        lastName={lastName}
+        avatar={avatar}
+        handleConfirm={handleConfirm}
+      />
+      <ConfirmLogOut
```
Comment (k-krasilnikova) [inline frontend/src/components/Layout/MobileMenu/MobileMenuContainer.tsx:60]: Rename it please to the `ConfirmLogOutModalWindow`

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/components/Layout/Header/styled.ts [hunk 3]
```diff
@@ -13,90 +28,217 @@ export const LayoutHeader = styled(Grid)({
...
+    margin: '8px 20px 8px 0px',
   },
   [theme.breakpoints.up('xl')]: {
-    maxWidth: '730px!important',
+    height: '50px',
+    fontSize: '24px',
+    margin: '15px 20px 15px 0px',
   },
-});
+  '@media(min-width: 1530px)': {
```
Comment (NikitaKhadnevich) [inline frontend/src/components/Layout/Header/styled.ts:153]: We have problem with "jump down" search menu on this and nearby resolutions(1528-1555px). Maybe will fix `width` of search block?

File: frontend/src/components/Layout/MobileMenu/styled.ts [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  backgroundColor: theme.palette.secondary.main,
+  padding: '8px 16px 40px 8px',
+  position: 'absolute',
+  top: HEADER_HEIGHT_MOBILE,
+  right: '0px',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'flex-start',
+});
+
```
Comment (NikitaKhadnevich) [inline frontend/src/components/Layout/MobileMenu/styled.ts:28]: We have problem with this item, i am not sure that its concern for this component, but we have little problem on 375px
[https://ibb.co/4RrRwG7](url) - Lets see

File: frontend/src/components/Layout/MobileMenu/styled.ts [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    padding: '0px',
+    '& span': {
+      fontFamily: 'Ubuntu, sans-serif',
+      fontWeight: 400,
+      fontSize: '16px',
+      paddingLeft: '2px',
+    },
+    '& svg': {
+      width: '50px',
+      height: '50px',
```
Comment (NikitaKhadnevich) [inline frontend/src/components/Layout/MobileMenu/styled.ts:127]: I had seen that icons are larger than on mockup. Is actual our version in this file?

**Review by NikitaKhadnevich — CHANGES_REQUESTED** [review]
Lets fix this wrongs

File: frontend/src/components/Layout/MobileMenu/styled.ts (hunk 4 above)
Comment (Julia-Maliutsina) [reply frontend/src/components/Layout/MobileMenu/styled.ts:28]: https://prnt.sc/26qtilp

File: frontend/src/components/Layout/MobileMenu/styled.ts (hunk 5 above)
Comment (Julia-Maliutsina) [reply frontend/src/components/Layout/MobileMenu/styled.ts:127]: fixed

File: frontend/src/components/Layout/Header/styled.ts (hunk 3 above)
Comment (Julia-Maliutsina) [reply frontend/src/components/Layout/Header/styled.ts:153]: fixed

File: frontend/src/components/Layout/MobileMenu/MobileMenu.tsx (hunk 1 above)
Comment (Julia-Maliutsina) [reply frontend/src/components/Layout/MobileMenu/MobileMenu.tsx:92]: Fixed

**Pull request merged**